TwitchLib.Client/TwitchClient.cs

```diff
@@ -383,17 +383,17 @@ public class TwitchClient : ITwitchClient
         /// <summary>
         /// Fires when TwitchClient attempts to host a channel it is in.
         /// </summary>
-        public event EventHandler OnSelfRaidError;
+        public event AsyncEventHandler<OnSelfRaidErrorArgs> OnSelfRaidError;
 
         /// <summary>
         /// Fires when TwitchClient receives generic no permission error from Twitch.
         /// </summary>
-        public event EventHandler OnNoPermissionError;
+        public event AsyncEventHandler<OnNoPermissionErrorArgs> OnNoPermissionError;
 
         /// <summary>
         /// Fires when newly raided channel is mature audience only.
         /// </summary>
-        public event EventHandler OnRaidedChannelIsMatureAudience;
+        public event AsyncEventHandler<OnRaidedChannelIsMatureAudienceArgs> OnRaidedChannelIsMatureAudience;
```

Comment:
It seems that those events are missing in `ITwitchClient` interface. Could you add them?

Reply:
The interface is completely out of sync, will fix.